fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -325,7 +326,7 @@ def implementation
         implm += "let args: [RubyCommand.Argument] = []\n"
       else
         implm += "let args = [#{args.group_by { |h| h[:name] }.keys.join(",\n")}]\n"
-        implm += ".compactMap { $0 }\n"
+        implm += ".compactMap { $0?.nilify(when: { $0.value == nil }) }\n"
```

Comment:
```suggestion
        implm += ".filter { $0?.value != nil } \n"
```

What about this? Isn't this simpler?

Reply:
🤦‍♂️ I'm an idiot

Reply:
Wasn't exactly doing that, but gave me the idea to get rid of the `nilify` method ❤️